odoo/tools/convert.py

```diff
@@ -376,10 +376,11 @@ def _tag_record(self, rec, extra_vals=None):
                 return None
             # else create it normally
 
+        foreign_record_to_create = False
         if xid and xid.partition('.')[0] != self.module:
             # updating a record created by another module
             record = self.env['ir.model.data']._load_xmlid(xid)
-            if not record:
+            if not record and not (foreign_record_to_create := nodeattr2bool(rec, 'forcecreate')):  # Allow foreign records if explicitely stated
```

Comment:
I know we talked about it with AL and ODO, but did this get reviewed by the ORM?

This solution is still the dirtiest IMO :shrug: